src/olympia/constants/promoted.py

```diff
@@ -56,7 +56,6 @@ def __bool__(self):
     search_ranking_bump=500,  # TODO: confirm this bump
     warning=False,
     pre_review=True,
-    admin_review=True,
```

Comment:
apparently it _doesn't_ require admin review according to https://docs.google.com/spreadsheets/d/1ENbv1CuLDd25Sk0XAwgbWTl9is5rV12OxJD6g1kUi48/edit#gid=1788006942